src/braingeneers/utils/common_utils_test.py

```diff
@@ -1,8 +1,10 @@
 import unittest
 from unittest.mock import patch, MagicMock
-import common_utils  # Updated import statement
+from common_utils import checkout, checkin, force_release_checkout
```

Comment:
Also evidence that this test is not running.

Reply:
Yeah, we should fix that. I'd leave that to you if it's ok, but I would suggest putting that on your backlog behind the workflow tasks. Automated unit tests are quite valuable long run, for sure, but we have a number of people backlogged on the workflow automation at the moment, so that's higher priority. I'll fix this before re-submitting, but I think we can merge this without the automated unit tests.